src/main/java/org/opentripplanner/standalone/config/RouterConfig.java

```diff
@@ -108,18 +111,45 @@ The fields separated by whitespace are (in order):
           .summary("Configuration for the Transmodel GraphQL API.")
           .asObject()
       );
+    Duration streetRoutingTimeout = parseStreetRoutingTimeout(
+      root,
+      StreetPreferences.DEFAULT.routingTimeout()
+    );
     this.routingRequestDefaults =
-      RouteRequestConfig.mapDefaultRouteRequest(
-        root,
-        "routingDefaults",
-        parseStreetRoutingTimeout(root, StreetPreferences.DEFAULT.routingTimeout())
-      );
+      RouteRequestConfig.mapDefaultRouteRequest(root, "routingDefaults", streetRoutingTimeout);
     this.transitConfig = new TransitRoutingConfig("transit", root, routingRequestDefaults);
     this.updatersParameters = new UpdatersConfig(root);
     this.rideHailingConfig = new RideHailingServicesConfig(root);
     this.vectorTileLayers = VectorTileConfig.mapVectorTilesParameters(root, "vectorTileLayers");
     this.flexConfig = new FlexConfig(root, "flex");
 
+    this.apiProcessingTimeout =
+      root
+        .of("apiProcessingTimeout")
+        .since(OtpVersion.V2_3)
```

Comment:
Now that 2.3 has been released this needs to be updated.

Reply:
done